.circleci/dist_compile.yml

```diff
@@ -352,6 +352,7 @@ jobs:
       simdjson_SOURCE: BUNDLED
       xsimd_SOURCE: BUNDLED
       DuckDB_SOURCE: BUNDLED
+      Boost_SOURCE: BUNDLED
```

Comment:
These changes wont work as the Scheduled jobs are run in github actions (not circle ci) , i.e make changes in .github/workflows/scheduled.yml.